src/worker/token.worker.ts

```diff
@@ -31,7 +50,9 @@ const messageHandler = async ({
   const { audience, scope } = auth || {};
 
   try {
-    const body = JSON.parse(fetchOptions.body);
+    const body = useFormData
+      ? parseFormData(fetchOptions.body)
```

Comment:
Since you can assume no IE11, you can use `URLSearchParams` to deserialize the form data

Also, if you did the serialization much later than [here](https://github.com/auth0/auth0-spa-js/blob/6740c33fc504d706d8b32d414fbc50eb9d13ed1f/src/api.ts#L26-L28) ([here](https://github.com/auth0/auth0-spa-js/blob/6740c33fc504d706d8b32d414fbc50eb9d13ed1f/src/http.ts#L90) for example) - you wouldn't need to `serialize to form -> deserialize to object -> serialize to form` rather just do `serialize to form` once (optional suggestion since I know we were already serializing it and deserializing it when it was JSON only)

Reply:
I'll leave that for a future improvement in another related PR, as I've identified a couple other bits I'd like to tweak as well.